docker/solr/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM solr:7.7.3-slim
+USER 0
+RUN apt-get update  \
+    && apt-get install git -y  \
+    && apt-get install rsync -y  \
+    && apt-get install curl -y  \
+    && apt-get install nano -y  \
+    && apt-get clean \
+    && git clone https://github.com/europeana/search
+COPY solr-schema.sh /opt/solr/search
+COPY europeana-collection-aliasing-solr6.6.5-0.0.1-SNAPSHOT.jar /opt/solr/contrib/lib/europeana-collection-aliasing-solr6.6.5-0.0.1-SNAPSHOT.jar
```

Comment:
Where is this file supposed to be? I don't have it.

Reply:
this goes here `/opt/solr/contrib/lib/` and is used by solr. This I found about it when replicating solr in the local environment, and obtained it from solr 7 server. Infra Team provided it.

Reply:
yeah but do we need to have the file elsewhere? How can someone cloning this project run the docker, I don't have the file.

Reply:
When cloning the repository the file is downloaded to the local folder, and when building the image the file is copied to the docker image so you will have it there.

Reply:
You said you obtained if from A server, now you are saying it is in the search repo yet I don't see it.

Reply:
done. it should be there now.